proposed/extended-coding-style-guide.md

```diff
@@ -546,6 +546,15 @@ class ReturnTypeVariations
 }
 ~~~
 
+Variadic argument three dots MUST NOT have space before argument name:
+
+```php
+public function process(string $algorithm, ...$parts)
+{
+    // processing
+}
+```
+
```

Comment:
~~~suggestion

When combining both the reference operator and the variadics three dots, there MUST NOT be any space between the two of them:

```php
public function process(string $algorithm, &...$parts)
{
    // processing
}
```

~~~